docs/conf.py

```diff
@@ -205,8 +206,8 @@
     "https://sourceforge.net/projects/vcxsrv/",
     "https://sourceforge.net/projects/xming/",
     "http://www.straightrunning.com/XmingNotes/",
-    "https://unix.stackexchange.com", # it seems stackexchange is now blocking bots
-    ]
+    "https://unix.stackexchange.com",  # it seems stackexchange is now blocking bots
+]
```

Comment:
We can skip it now, but in the future please consider a separate PR for such drive-by changes (especially when they outweigh intended changes).